frontend/src/modules/application/components/FormFields.jsx

```diff
@@ -1,29 +1,44 @@
+// This file contains all of the fields for the application form.  
```

Comment:
I like the explanation, but please convert this to a multiline comment for readability. `{/* A JSX comment */}`